### ISSUE TITLE
Multi-line arrays are not auto-indented correctly

### ISSUE THREAD

**Issue by aaronjensen** [gandm/language-babel#32]
Currently it looks like this:

```js
  mixins: [
  Stores.CurrentUser.mixin,
  Stores.CoachParticipants.mixin,
  Stores.Errors.mixin,
  Stores.LastViewed.mixin,
  Stores.ParticipantConversations.mixin,
  Stores.ParticipantGoal.mixin,
  Stores.ParticipantSuggestions.mixin,
  Stores.PendingUpdate.mixin,
  RouteHandlerKey
  ],
```

I expect it to look like this (how https://github.com/atom/language-javascript indents them):

```js
  mixins: [
    Stores.CurrentUser.mixin,
    Stores.CoachParticipants.mixin,
    Stores.Errors.mixin,
    Stores.LastViewed.mixin,
    Stores.ParticipantConversations.mixin,
    Stores.ParticipantGoal.mixin,
    Stores.ParticipantSuggestions.mixin,
    Stores.PendingUpdate.mixin,
    RouteHandlerKey
  ],
```

**Comment by gandm** [comment]
I can't replicate this. 
![indent](https://cloud.githubusercontent.com/assets/2313237/8873641/592f8290-3200-11e5-9849-0766430eff32.gif)
.

**Comment by aaronjensen** [comment]
It looks like it is the [react](https://github.com/orktes/atom-react) package that is causing it. It works when that is disabled. I'll open an issue there.

**Comment by aaronjensen** [comment]
This is fixed in react@0.12.6
